staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -433,6 +434,9 @@ func (c *Config) Complete(informers informers.SharedInformerFactory) CompletedCo
 	}
 
 	AuthorizeClientBearerToken(c.LoopbackClientConfig, &c.Authentication, &c.Authorization)
```

Comment:
`AuthorizeClientBearerToken` is used in multiple places with distinct `Complete` flows.  The only way to keep it consistently working (in the sense of adding the authenticated group) is to wrap the `tokenAuthenticator := authenticatorfactory.NewFromTokens(tokens)` in `AuthorizeClientBearerToken` with a `group.NewAuthenticatedGroupAdder`.

`pkg/kubeapiserver/authenticator/config.go` and `staging/src/k8s.io/apiserver/pkg/server/config.go` should be left unchanged.